modules/puppetca/puppetca_main.rb

```diff
@@ -31,10 +31,9 @@ def disable certname
         end
       end.uniq.compact
       if found
-        autosign = File.open(autosign_file, File::TRUNC|File::RDWR)
-        autosign.write entries.join("\n")
-        autosign.write "\n"
-        autosign.close
+        open(autosign_file, File::TRUNC|File::RDWR) do |autosign|
+          autosign.write entries.join
```

Comment:
For my understanding: does this mean there is no more newline at the end of the file?